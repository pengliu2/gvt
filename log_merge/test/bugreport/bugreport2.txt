------ SYSTEM LOG (logcat -v threadtime -d *:v) ------
--------- beginning of /dev/log/main
10-13 07:54:36.796  1371  1371 D PowerManager: Partial WakeLock: Acquired TAG: PollingManager isRefCounted: true Ref Count: 1 PID: 1371 UID: 10077 Held: true
------ VM TRACES JUST NOW (/data/anr/traces.txt.bugreport: 2010-10-13 08:40:04) ------
