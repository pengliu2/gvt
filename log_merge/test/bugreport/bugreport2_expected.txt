SYSTEM LOG:--------- beginning of /dev/log/main
SYSTEM LOG:10-13 07:54:36.796  1371  1371 D PowerManager: Partial WakeLock: Acquired TAG: PollingManager isRefCounted: true Ref Count: 1 PID: 1371 UID: 10077 Held: true
